### PR TITLE
Joel db2 related to issue 275

### PR DESCRIPTION
**Pull request by jgod1360**
https://github.ibm.com/IBM-DEG/Ecosystem-Technical-Enablement/issues/275

Test results: https://cloud.ibm.com/schematics/workspaces/us-east.workspace.joel-db2-related-to-275-branch-testing-2.9cf3ff88/jobs?region=us

### CODE REVIEW

File: modules/Db2/scripts/uninstall_Db2.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+eval "$(jq -r '@sh "export KUBECONFIG=\(.kubeconfig) DB2_PROJECT_NAME=\(.db2_project_name)"')"
+
+ibmcloud login -q -apikey "${API_KEY}"
+ibmcloud ks cluster config -c "${CLUSTER_ID}" --admin
+
+echo "Deleting Resources"
```
Comment (annumberhocker) [inline modules/Db2/scripts/uninstall_Db2.sh:6]: These are still resources for CP4I, not CP4BA

**Review by annumberhocker — CHANGES_REQUESTED** [review]
Almost there. Still need to fix the triggers for the uninstall and the uninstall_db2.sh script.

File: modules/Db2/scripts/uninstall_Db2.sh (hunk 1 above)
Comment (jgod1360) [reply modules/Db2/scripts/uninstall_Db2.sh]: Fixed

File: modules/Db2/main.tf [hunk 2]
```diff
@@ -57,18 +70,31 @@ resource "null_resource" "install_db2" {
...
     }
   }
+
+  provisioner "local-exec" {
+    when        = destroy
+    command     = "./uninstall_db2.sh"
+    working_dir = "${path.module}/scripts"
+
+    environment = {
+      kubeconfig        = self.triggers.kubeconfig
```
[the remark this answers is not in the document]
Comment (jgod1360) [reply modules/Db2/main.tf]: Fixed

**Review by annumberhocker — APPROVED** [review]
LGTM

**Pull request merged**